examples/07-EMIT/interference_classification.py

```diff
@@ -3,32 +3,35 @@
 import pyaedt
 import sys
 
-def interference_classification(tx_radios, rx_radios, radios, emitapp, rev, domain, use_filter, filter):
+def interference_classification(emitapp, use_filter, filter):
     """
     Classisify interference type as according to inband/inband, 
     out of band/in band, inband/out of band, and out of band/out of band.
 
     Parameters
-        ----------
-        tx_radios : string list of tx radio names
-        rx_radios : string list of rx radio names
-        radios : list of radio objects
+    ----------
         emitapp : instance of Emit
-        rev : emitapp results
-        domain : interaction domain of emit results
         use_filter : boolean value, True if filtering is being used
```

Comment:
```suggestion
        use_filter : bool, optional
            Whether filtering is being used. The default is ``False``.
```
I wasn't sure how to edit the first parameter to fit our PyAnsys docstring style.